### PR TITLE
(DNM) Fixing ts diffing script

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 🧢 Your Project:

RSP

### CODE REVIEW

File: packages/@react-spectrum/dnd/package.json [hunk 1]
```diff
@@ -33,10 +33,7 @@
   "dependencies": {
     "@babel/runtime": "^7.6.2",
     "@react-aria/dnd": "3.0.0-alpha.5",
-    "@react-aria/utils": "^3.0.0",
-    "@react-spectrum/utils": "^3.0.0",
-    "@react-stately/dnd": "3.0.0-alpha.4",
-    "@react-types/shared": "^3.0.0"
+    "@react-stately/dnd": "3.0.0-alpha.4"
```
Comment (snowystinger) [inline packages/@react-spectrum/dnd/package.json:36]: ? should this be in a different PR?

**Review by snowystinger — COMMENTED** [review]
one question, otherwise looks good

**Review by snowystinger — APPROVED** [review]

**Comment by LFDanLu** [comment]
complains about ` Cannot copy '../../../../../node_modules/@babel/parser/bin/babel-parser.js' to a subdirectory of itself, '../../../../../node_modules/@babel/parser/bin/babel-parser.js'.` in a subsequent run, to investigate later

File: scripts/buildPublishedAPI.js [hunk 2]
```diff
@@ -124,13 +124,18 @@ async function build() {
   let babelPatch = patches.find(name => name.startsWith('@babel'));
   fs.copySync(path.join(__dirname, '..', 'patches', babelPatch), path.join(dir, 'patches', babelPatch));
 
-  // Copy package.json for each package into docs dir so we can find the correct version numbers
+  // Copy from node_modules and pretend it is src
   console.log('moving over from node_modules');
   for (let p of packages) {
     if (!p.includes('spectrum-css') && fs.existsSync(path.join(dir, 'node_modules', p))) {
+      let json = JSON.parse(fs.readFileSync(path.join(dir, 'node_modules', p)), 'utf8');
```
Comment (ktabors) [inline scripts/buildPublishedAPI.js:131]: I asked Daniel about why this is reading from `node_modules` instead of `packages` like below. He said it was an order of operations issue and existed in `node_modules` at this point of the code. My current understanding is the important change is below where we remove `nightly` builds from ts-differ.

**Comment by LFDanLu** [comment]
Will pick up again later
